caddyhttp/httpserver/replacer.go

```diff
@@ -431,6 +434,68 @@ func (r *replacer) getSubstitution(key string) string {
 			return "UNKNOWN" // this should never happen, but guard in case
 		}
 		return r.emptyValue
+	case "{tls_client_escaped_cert}":
+		if r.request.TLS != nil && len(r.request.TLS.PeerCertificates) > 0 {
```

Comment:
Can we pull out this section of code into a separate function.

    		if r.request.TLS != nil && len(r.request.TLS.PeerCertificates) > 0 {
			cert := r.request.TLS.PeerCertificates[0]

Reply:
Done.